### PR TITLE
Add new 6.0 packages and binary distributions in supported platforms

### LINKED ISSUE #2610: Document new binary releases
### Description
I've added support for several new tarball releases for 6.0. These include:

* `freebsd/arm` (sensu-agent & sensuctl)
* `linux/s390x` (sensu-agent & sensuctl)
* `linux/ppc64le` (sensu-agent, sensu-backend & sensuctl)

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [x] new doc (requires a completely new doc)
- [ ] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
* Downloads page
* Supported Platforms https://docs.sensu.io/sensu-go/latest/platforms/

### LINKED ISSUE #2611: Document new package architectures
### Description
We're shipping deb & rpm packages for additional Linux architectures with the 6.0 release. See https://github.com/sensu/sensu-enterprise-go/pull/1147 for a list of the new architectures and the list of repositories we'll be publishing each package to.

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [x] new doc (requires a completely new doc)
- [ ] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
* Downloads page
* Supported Platforms https://docs.sensu.io/sensu-go/latest/platforms/

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- Adds freebsd/arm and linux/s390x and linux/ppc64le with placeholder links to Linux table in supported platforms doc.
- Adds list of new supported packages in supported platforms doc.

Need to make sure the links will be correct when 6.0 is released.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2610
Closes https://github.com/sensu/sensu-docs/issues/2611

### CODE REVIEW

File: content/sensu-go/6.0/platforms.md [hunk 1]
```diff
@@ -326,25 +331,26 @@ To build Sensu Go from source, see the [contributing guide on GitHub][16].
...
-[34]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.21.0/sensu-go_5.21.0_freebsd_386.tar.gz
-[35]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.21.0/sensu-go_5.21.0_freebsd_386.zip
-[36]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.21.0/sensu-go_5.21.0_solaris_amd64.tar.gz
-[37]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.21.0/sensu-go_5.21.0_solaris_amd64.zip
-[54]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.21.0/sensu-go_5.21.0_linux_amd64.tar.gz
-[55]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.21.0/sensu-go_5.21.0_linux_arm64.tar.gz
-[56]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.21.0/sensu-go_5.21.0_linux_armv5.tar.gz
-[57]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.21.0/sensu-go_5.21.0_linux_armv6.tar.gz
+[18]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/6.0.0/sensu-go_6.0.0_linux_armv7.tar.gz
+[19]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/6.0.0/sensu-go_6.0.0_freebsd_arm.tar.gz
```
Comment (amdprophet) [inline content/sensu-go/6.0/platforms.md]: My mistake. This should be expanded to `freebsd_arm_5`, `freebsd_arm_6`, and `freebsd_arm_7`.

**Review by amdprophet — CHANGES_REQUESTED** [review]

**Comment by amdprophet** [comment]
Here's the naming of the new files:
Screen Shot 2020-07-16 at 1 06 25 AM
Screen Shot 2020-07-16 at 1 06 07 AM

**Comment by hillaryfraley** [comment]
@amdprophet Do the file names mean that these new packages/binaries are added for 5.21 in addition to 6.0? Or the file names just aren't updated for 6.0 yet?

Edit: is the first part of the link still `https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.21.0/` as for other binaries? For example, is this correct for the complete link for freebsd armv5:
`https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.21.0/sensu-enterprise-go_5.21.0_freebsd_armv5.tar.gz`

**Comment by hillaryfraley** [comment]
I'm not sure I love the way I redid the packages tables, but I couldn't think of a way around it.

**Comment by hillaryfraley** [comment]
I think the agent and sensuctl package tables look a little better...

**Comment by amdprophet** [comment]
> @amdprophet Do the file names mean that these new packages/binaries are added for 5.21 in addition to 6.0? Or the file names just aren't updated for 6.0 yet?
> 
> Edit: is the first part of the link still `https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.21.0/` as for other binaries? For example, is this correct for the complete link for freebsd armv5:
> `https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.21.0/sensu-enterprise-go_5.21.0_freebsd_armv5.tar.gz`

Just 6.0. I'll have to verify what the final URLs will be; they may change slightly.

**Comment by amdprophet** [comment]
The syntax of the link you provided in the example should be correct.

**Review by amdprophet — APPROVED** [review]

**Pull request merged**